vue/src/TagmanagerTrackingCode/TrackingCodeCommon.vue

```diff
@@ -383,6 +407,34 @@ export default defineComponent({
       }
       return '';
     },
+    getAdvancedStepText() {
+      const stepText = translate('TagManager_SelectContainerForWebsite', '<strong>', '</strong>');
+      if (this.idContainer) {
+        return stepText;
+      }
+
+      // If not container is found, we should include a link to the manage containers area
+      const manageContainerURL = this.linkTo('manageContainers', String(this.site?.id), '');
+      const manageContainersText = translate(
+        'TagManager_ManageContainersLink',
+        `<a href="${manageContainerURL}" target="_blank" rel="noreferrer noopener">`,
+        '</a>',
+      );
+
+      return `${stepText} ${manageContainersText}`;
+    },
+    getAdvancedStepInfo() {
+      const idSite = this.site && this.site.id ? this.site.id as string : '';
+      const link = this.linkTo('dashboard', idSite, this.idContainer, []);
+
+      return translate(
+        'TagManager_CustomiseContainer',
+        `<a href="${link}">`,
```

Comment:
@snake14 should we specify target="_blank" as it opens the URL in the same tab instead of new tab

Reply:
This isn't new code, it's just been moved from a different Vue component. I didn't want to change the UX.